### PR TITLE
feat: Cross-platform developer tooling post

### PR DESCRIPTION
**Pull request by jimmidyson**

### CODE REVIEW

File: content/posts/a-journey-to-cross-platform-developer-tooling/index.md [hunk 1]
```diff
@@ -0,0 +1,315 @@
...
+date: 2023-07-19T10:39:28+01:00
+featured: false
+tags: ["development"]
+feature_image: feature.webp
+excerpt: Streamlining the developer experience
+---
+
+Over the 25 or so years of being a software engineer, I've worked on many projects for quite a few different companies. At the start of my career, developer tooling was pretty simple. I don't mean basic (funny, considering I did do quite a bit of Visual Basic development at the start of my career), I mean simple: there were few options and those options that were available were generally dictated to developers via company policy and prescribed OS builds.
+
+Jump forward to now and now the options for developer tooling are pretty much endless. This is a good thing, it has enabled developers find their own ways to be as productive as possible. IDEs are a good example. The [Vim][] vs [Emacs][] debate will likely never go away, but it definitely feels quieter than ever before because it doesn't matter so much. I didn't think I would ever move away from [Vim][] personally, but nowadays I'm a [VS Code][] convert. I also have colleagues that prefer to work in [Goland by Jetbrains][].
```
Comment (dkoshkin) [inline content/posts/a-journey-to-cross-platform-developer-tooling/index.md:13]: Will you be putting a link to all of these?

**Review by dkoshkin — APPROVED** [review]
Great post! When are we switching to Devbox :)

File: content/posts/a-journey-to-cross-platform-developer-tooling/index.md (hunk 1 above)
Comment (jimmidyson) [reply content/posts/a-journey-to-cross-platform-developer-tooling/index.md:13]: These are linked already, this is how you can use reference links when the reference matches the actual text.

**Comment by jimmidyson** [comment]
@dkoshkin We're already using devbox in mindthegap btw 😜

File: content/posts/a-journey-to-cross-platform-developer-tooling/index.md (hunk 1 above)
Comment (dkoshkin) [reply content/posts/a-journey-to-cross-platform-developer-tooling/index.md:13]: TIL!

**Review by mstruebing — APPROVED** [review]
Love it! 🎉

File: content/posts/a-journey-to-cross-platform-developer-tooling/index.md [hunk 2]
```diff
@@ -0,0 +1,403 @@
...
+authors: ["jdyson"]
+title: "A Long Journey to Cross Platform Developer Tooling Utopia (For Now)"
+date: 2023-07-19T10:39:28+01:00
+featured: false
+tags: ["development"]
+feature_image: feature.webp
+excerpt: Streamlining the developer experience.
+---
+
+Over the 25 or so years of being a software engineer, I've worked on many projects for quite a few different companies.
```
Comment (tillt) [inline content/posts/a-journey-to-cross-platform-developer-tooling/index.md:11]: What a great stage setup this is -- love this historical context a lot.

**Review by tillt — APPROVED** [review]
Beautiful post that got me fired up for devbox. TIL I learned about `webp` -- totally passed by me until today.

**Pull request merged**